weatherAplication/Modules/Search/View/SearchViewController.swift

```diff
@@ -77,7 +77,7 @@ extension SearchViewController: UITableViewDelegate, UITableViewDataSource {
 
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         let cityInfoToSend = viewModel.cityInfoArray[indexPath.row]
```

Comment:
tutaj chodziło mi o to żebyś wysyłał do ViewModelu sam index path na zasadzie viewModel.cellPressed(index: indexPath) ew. indexPath.row. Potem ViewModel już u siebie odpowiada za wyciągnięcie z arraya cityInfoTosend bo to w nim powinna znaleźć się logika wyciągania danych